tools/Status/Status.py

```diff
@@ -213,13 +216,16 @@ def _format(field: str, value: Any, state_styles: dict) -> str:
 # is the human-readable name. These are sometimes the same
 AVAILABLE_COLUMNS = {
     "State": "State",
+    "Partition": "Queue",
     "End": "End",
     "User": "User",
     "JobID": "JobID",
     "JobName": "JobName",
+    "SegmentId": "Segment",
     "Elapsed": "Elapsed",
-    "NCPUS": "Cores",
-    "NNodes": "Nodes",
+    "NCPUS": "NumCores",
+    "NNodes": "NumNodes",
```

Comment:
Note that there's limited horizontal space in the terminal and the "Num" prefix makes the col name a bit longer. maybe keep the shorter "Cores" and "Nodes" and make the one below "NodeList" like in Slurm? Cores and nodes are short integers so the col width is set by the col name, whereas "NodeList" is longer so a longer col name doesn't limit the col width.